pylintrc

```diff
@@ -7,7 +7,9 @@ disable=
  C0415, # Import outside toplevel
  E1101, # no-member
  R0904, # Too many public methods
+ R0903, # Too few public methods
  R1702, # Too many nested blocks
+ R0901, # Too many ancestors
```

Comment:
is it about Provider  ? maybe we should just put ignorance there ?

Reply:
> is it about Provider ? maybe we should just put ignorance there ?

FilteredSingleTableView in ocw/views.py